testutil/ciao-down/workloads/clearcontainers-3.x.yaml

```diff
@@ -0,0 +1,249 @@
+---
+base_image_url: https://cloud-images.ubuntu.com/xenial/current/xenial-server-cloudimg-amd64-disk1.img
+base_image_name: Ubuntu 16.04
+hostname: singlevm
+needs_nested_vm: true
+vm:
+{{with .GoPath}}
+  mounts:
+  - tag: hostgo
+    security_model: none
+    path: {{.}}
+{{end}}
+...
+---
+{{- define "ENV" -}}
+{{proxyVars .}}
+{{- print " DEBIAN_FRONTEND=noninteractive DEBCONF_NONINTERACTIVE_SEEN=true " -}}
+{{end}}
+{{ define "GOPATH" }}{{with .GoPath}}{{$.MountPath "hostgo"}}{{else}}/home/{{.User}}/go{{end}}{{end}}
+#cloud-config
+write_files:
+{{- if len $.HTTPProxy }}
+ - content: |
+     [Service]
+     Environment="HTTP_PROXY={{$.HTTPProxy}}"{{if len .HTTPSProxy}} "HTTPS_PROXY={{.HTTPSProxy}}{{end}}"{{if len .NoProxy}} "NO_PROXY={{.NoProxy}},{{.Hostname}}{{end}}"
+   path: /etc/systemd/system/docker.service.d/http-proxy.conf
+{{- end}}
+{{with proxyEnv . 5}}
+ - content: |
+{{.}}
+   path: /etc/environment
+{{end}}
+ - content: |
+     [Service]
+     ExecStart=
+     ExecStart=/usr/bin/dockerd -D --add-runtime cor=/usr/bin/cc-runtime --default-runtime=cor
+   path: /etc/systemd/system/docker.service.d/clr-containers.conf
+ - content: |
+     #!/bin/sh
+     printf "\n"
+     printf "\n"
+     printf "Your go code is at {{template "GOPATH" .}}\n"
+     printf "You can also edit your code on your host system \n"
+     printf "To build cc-runtime from sources \n"
+     printf "go get -d github.com/clearcontainers/runtime... \n"
+     printf "cd {{template "GOPATH" .}}/src/github.com/clearcontainers/runtime \n"
+     printf "make build-cc-system\n"
+     printf "sudo -E PATH=$PATH make install-cc-system \n"
+     printf "\n"
+     printf "\n"
+   path: /etc/update-motd.d/10-ciao-help-text
+   permissions: '0755'
+ - content: |
+     [Unit]
+      Description=OCI-based implementation of Kubernetes Container Runtime Interface
+      Documentation=https://github.com/kubernetes-incubator/cri-o
+
+      [Service]
+      ExecStart=/usr/bin/crio --debug
+      Environment="HTTP_PROXY={{$.HTTPProxy}}"{{if len .HTTPSProxy}} "HTTPS_PROXY={{.HTTPSProxy}}{{end}}"{{if len .NoProxy}} "NO_PROXY={{.NoProxy}},{{.Hostname}}{{end}}"
```

Comment:
@markdryan @mcastelino  -- any suggestions on how to assign the VMs IP address here for no proxy?